tools/klee/main.cpp

```diff
@@ -422,6 +422,7 @@ void KleeHandler::processTestCase(const ExecutionState &state,
                                   const char *errorSuffix) {
   if (errorMessage && ExitOnError) {
     llvm::errs() << "EXITING ON ERROR:\n" << errorMessage << "\n";
```

Comment:
Not related with this PR, shouldn't we use `klee_error` here? Since you're already modifying the method..